momento-sdk/src/main/java/momento/sdk/messages/CacheDeleteResponse.java

```diff
@@ -13,10 +14,10 @@ class Success implements CacheDeleteResponse {}
    * thrown, or the cause of the error can be retrieved with {@link #getCause()}. The message is a
    * copy of the message of the cause.
    */
-  class Error extends SdkException implements CacheDeleteResponse {
+  class Error extends WrappedSdkException implements CacheDeleteResponse {
 
```

Comment:
nit: I'm not 100% sure where we have landed on this across all of the SDKs, but I had been advocating for exposing a property/method called `errorCode` on these, which would provide access to the actual enum of all of our different error types.  That way people could write switch statements that key off of the enum rather than having to do type checking on the exception object. See:

https://github.com/momentohq/standards-and-practices/blob/main/docs/client-specifications/error-handling.md#error-types

idk if we've stuck with this through all of the SDKs so I'm open to discussion on it, but that was my original hope/intent.

@pgautier404 it might be good for you to review this PR and weigh in on state-of-the-art.

Reply:
Yes, we should definitely be using an errorCode property to match against a public enum of the available codes listed in that standards and practices doc. You might take a look at [the nodejs implementation](https://github.com/momentohq/client-sdk-nodejs/blob/main/src/errors/errors.ts), which I think makes the pattern pretty clear. Let me know if it would be useful to take a look at this and/or implementations in other SDKs together, @nand4011.

Reply:
I didn't implement the error codes because they are 1:1 with exceptions. I have no problem adding them in though.